pom.xml

```diff
@@ -150,6 +151,32 @@
             </build>
         </profile>
 
+        <profile>
+            <id>spotbugs</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.github.spotbugs</groupId>
+                        <artifactId>spotbugs-maven-plugin</artifactId>
+                        <version>${spotbugs-maven-plugin.version}</version>
+                        <executions>
+                            <execution>
+                                <phase>compile</phase>
+                                <goals>
+                                    <goal>check</goal>
```

Comment:
Can you please update the readme how to use it via mvn command?

Reply:
Done 👍